### PR TITLE
🐛 Failing test on cypress/task

### PR DESCRIPTION
**Pull request by EmaSuriano**
Given the [new documentation for v3](https://docs.percy.io/docs/cypress#removing-tasks) the `require` of `@percy/cypress/task` is not required anymore, and it's also making the process failed.

This PR just removed the plugin file :)

### CODE REVIEW

**Comment by Robdel12** [comment]
Hey @EmaSuriano, thanks for the PR! Surprised CI doesn't fail over this.

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-// ***********************************************************
-
-// This function is called when a project is opened or re-opened (e.g. due to
-// the project's config changing)
-let percyHealthCheck = require("@percy/cypress/task");
-
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-  on("task", percyHealthCheck);
```
Comment (Robdel12) [inline cypress/plugins/index.js:18]: Instead of deleting the whole file, can you just remove this line & 13?

**Review by Robdel12 — CHANGES_REQUESTED** [review]

File: cypress/plugins/index.js (hunk 1 above)
Comment (EmaSuriano) [reply cypress/plugins/index.js:18]: Yes sure, so then it will be the same as the default cypress config.

**Comment by Robdel12** [comment]
Thanks for the PR! 😃

**Pull request merged**